unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/ControlsModule/SliderSettingsControlView.cs

```diff
@@ -9,17 +9,35 @@ public class SliderSettingsControlView : SettingsControlView
         [SerializeField] private Slider slider;
         [SerializeField] private TextMeshProUGUI indicatorLabel;
 
+        public Slider sliderControl { get => slider; }
+
         public override void Initialize(SettingsControlModel controlConfig, SettingsControlController settingsControlController)
         {
+            slider.maxValue = ((SliderControlModel)controlConfig).sliderMaxValue;
+            slider.minValue = ((SliderControlModel)controlConfig).sliderMinValue;
+            slider.wholeNumbers = ((SliderControlModel)controlConfig).sliderWholeNumbers;
```

Comment:
We can store the casted `SliderControlModel` in a temp variable for easier reading.

Reply:
Done in the feedback PR: https://github.com/decentraland/explorer/pull/1784 Thanks!